tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -1810,6 +1810,18 @@ class ParameterizedOpConverterTestBase
                      DataType tf_type, const std::vector<T>& values,
                      const std::vector<int32>& partial_input_shape_dims = {},
                      Status add_input_status = Status::OK()) {
+
```

Comment:
Can we remove this blank line? Clang format will complain about it and I will need to modify your PR to merge it. If I need to modify the PR, I will have to invite another person to review my changes, that is the step I want to avoid.

Reply:
Fixed